src/Corporation/data/Constants.ts

```diff
@@ -61,7 +61,7 @@ export const CorporationConstants: {
 
   ProductProductionCostRatio: 5, //Ratio of material cost of a product to its production cost
 
-  DividendMaxPercentage: 1,
+  DividendMaxRate: 1,
```

Comment:
would this default case still be 1? not 0.01?

Reply:
Nope, this is a case where the property was named percentage but was actually used by multiplying by 100 (e.g. https://github.com/danielyxie/bitburner/blob/4993a024211714c5af9a79fd73f8d9a37ed4025f/src/Corporation/Corporation.tsx#L129)